HealthcareSystem/Backend/Service/RoomAndEquipment/ConsumableEquipmentService.cs

```diff
@@ -9,33 +9,56 @@
 using Model.Users;
 using Repository;
 using System.Collections.Generic;
+using Backend.Service.RoomAndEquipment;
+using Backend.Repository;
+using Backend.Repository.EquipmentInRoomsRepository;
 
 namespace Service.RoomAndEquipment
```

Comment:
Fix indents

Reply:
Solved in 227dd73d685d299759a4b4d3dbf857be00385e68.